doc/releases/release-notes-3.6.rst

```diff
@@ -573,38 +607,185 @@ Networking
+    communication with host to start during boot.
+  * Added BLE TCAT implementation in OpenThread platform.
+  * Updated Crypto PSA backend for OpenThread with additional algorithms.
+  * Fixed ``otPlatAssertFail()``, so that it prints the location of the actual
+    assert instead of the function itself.
+
 * PPP:
 
+  * Fixed PPP connection termination when interface goes down.
+
+* Shell:
+
+  * Refactored networking shell module, so that instead of large single file, it
+    is split into submodules, at per command basis.
+  * Fixed unexpected timeout message when executing loopback ping.
+  * Added ``net sockets`` command to print information about open sockets and
+    socket services.
+  * Join IPv4/IPv6 multicast groups, if needed, when adding IPv4/IPv6 multicast
+    addresses via shell.
+  * Fixed ``tcp connect`` command operation (TCP context released prematurely).
+  * Added support for Echo option in telnet shell backend.
+  * Fixed unnecessary connection close in telnet shell backend in case of
+    non-fatal EAGAIN or ENOBUFS errors.
+  * Fixed double packet dereference in ping reply handler.
+  * Fixed possible deadlock when executing ``net arp`` command.
+  * Added more detailed Ethernet statistics printout for ``net stats`` command.
+  * Added ``net dhcpv4 server`` commands for DHCPv4 server management.
+  * Added shell module to manage TLS credentials.
+
 * Sockets:
 
-  * Added support for IPv4 multicast ``IP_ADD_MEMBERSHIP`` and ``IP_DROP_MEMBERSHIP`` socket options.
-  * Added support for IPv6 multicast ``IPV6_ADD_MEMBERSHIP`` and ``IPV6_DROP_MEMBERSHIP`` socket options.
+  * Added support for v4-mapping-to-v6, which allows IPv4 and IPv6 to share the
+    same port space.
+  * Added support for :c:macro:`IPV6_V6ONLY` socket option.
+  * Added support for :c:macro:`SO_ERROR` socket option.
+  * Fixed :c:func:`select` not setting ``writefds`` in case of errors.
+  * Added support for object core, which allows to track networks sockets and
+    their statistics.
+  * Added support for :c:func:`recvmsg`.
+  * Added support for :c:macro:`IP_PKTINFO` and :c:macro:`IPV6_RECVPKTINFO`
+    socket options.
+  * Added support for :c:macro:`IP_TTL` socket option.
+  * Added support for IPv4 multicast :c:macro:`IP_ADD_MEMBERSHIP` and
+    :c:macro:`IP_DROP_MEMBERSHIP` socket options.
+  * Added support for IPv6 multicast :c:macro:`IPV6_ADD_MEMBERSHIP` and
+    :c:macro:`IPV6_DROP_MEMBERSHIP` socket options.
+  * Improved doxygen documentation of BSD socket API.
+  * Fixed POLLERR error reporting in TLS sockets.
+  * Fixed DTLS handshake processing during :c:func:`poll`.
+  * Aligned DTLS socket :c:func:`connect` behavior with regular TLS (handshake
+    during connect call).
+  * Added Socket Services library, which allows to register multiple socket based
+    network services, and process them within a single thread.
+  * Added a new ``echo_service`` sample for Socket Services.
```

Comment:
@rlubos just spotted this other occurrence of "Services" where it should prob be "Service" -- see also line 804.

Reply:
Opened https://github.com/zephyrproject-rtos/zephyr/pull/68992 to address this.